jenkins/docker/Jenkinsfile

```diff
@@ -0,0 +1,83 @@
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS') 
+    }
+    agent none
+    stages {
+        stage('parameters') {
+            steps {
+                script {
+                    properties([
+                            parameters([
+                                    string(
+                                            defaultValue: 'https://github.com/opensearch-project/opensearch-build',
+                                            name: 'DOCKER_BUILD_GIT_REPOSITORY',
+                                            description: 'The git repository name that contains dockerfiles and the docker build script',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: 'main',
+                                            name: 'DOCKER_BUILD_GIT_REPOSITORY_REFERENCE',
+                                            description: 'The git reference (branch/tag/commit_id) of above repository',
+                                            trim: true
+                                    ),
+                                    string(
+                                            defaultValue: 'bash docker/ci/build-image-multi-arch.sh -v <TAG_NAME> -f <DOCKERFILE PATH>',
+                                            name: 'DOCKER_BUILD_SCRIPT_WITH_COMMANDS',
+                                            description: 'The script path of the docker build script, assuming you are already in root dir of DOCKER_BUILD_GIT_REPOSITORY',
+                                            trim: true
+                                    ),
+                                    booleanParam(
+                                            defaultValue: true,
```

Comment:
what is the difference between staging and prod here? https://hub.docker.com/u/opensearchstaging vs https://hub.docker.com/u/opensearchproject?

Reply:
Yes.

Reply:
Tweak description.